src/pages/NftCollections/List.tsx

```diff
@@ -206,12 +206,12 @@ export const ListOnDesktop: React.FC<{ isLoading: boolean; list: NFTCollection[]
                     )}
                     <Link
                       to={`/nft-collections/${typeHash || item.id}`}
-                      title={item.name}
+                      title={item.standard === 'spore' && item.creator === '' ? 'Unique items' : item.name}
```

Comment:
`item.standard === 'spore' && item.creator === '' ? 'Unique items' : item.name` can be assigned to a variable to avoid repeating.

Reply:
Refactor in new commit.